README.md

```diff
@@ -1 +1,3 @@
 # gha-test
```

Comment:
This is more of a safety check. After confirming the change request, you need to manually dismiss this review or resolve the conversation to get it merged